go.mod

```diff
@@ -335,7 +334,3 @@ replace github.com/prometheus/alertmanager => github.com/grafana/prometheus-aler
 // - https://github.com/grafana/franz-go/pull/3
 // - https://github.com/grafana/franz-go/pull/4
 replace github.com/twmb/franz-go => github.com/grafana/franz-go v0.0.0-20241009100846-782ba1442937
-
-// Pin Google GRPC to v1.65.0 as v1.66.0 has API changes and also potentially performance regressions.
-// Following https://github.com/grafana/dskit/pull/581
-replace google.golang.org/grpc => google.golang.org/grpc v1.65.0
```

Comment:
does it make sense to do a smaller jump first? we're skipping 5 minor versions and changing 10K lines in the underlying dependency. Can we do 1.66 in this PR and then week-by-week or so move up to 1.71?